cohort_4/playbook/ACTIVITIES.md

```diff
@@ -50,6 +54,63 @@
 It may be a good idea to play some music in the hackerspace while people are
 arriving.
 
+# Energizers
+
+## Where The West Wind Blows
+
+## Human Pictionary
+
+```
+Time: 10 min
+Materials: None
+Team Sizes: 
+  - when 4-8 people, group into teams of size 2
+  - when 9-12 people, group into teams of size 3
+  - when 13+ people, group into teams of size 4 or 5
+```
+
+**Facilitation:**
+
+- Organize the students get into teams.
+- Announce that there is no talking in this activity
+- Facilitator(s) will call out a random prompt (e.g. iPhone, giraffe, surfboard) and start the 30 second timer. 
+- Each team tries to represent that the prompt with their bodies in that time. 
+- At the end of the 30 seconds, everyone will freeze as the facilitator(s) goes around and judges the teams and makes funny and comments. 
+  - "This group is trying really hard not to fall. It really does look like th leaning tower of Pizza. Bonus points on being very angled."
+- After observing all the groups, the facilitator announces the winner.
+- The facilitator tells the winning team gets to pick, announce, and judge the next round.
+- You can repeat this for the duration you want this activity.
+
+## I Love My Neighbor Who ___
```

Comment:
What is the ___ for?

Perhaps `I Love My Neighbor Who...` might be a bit more clear?